File: samples/AdventureWorksLT.DbGen/ProductCategoryMockDb.cs
using DevZest.Data;

namespace DevZest.Samples.AdventureWorksLT
{
    public sealed class ProductCategoryMockDb : MockDb<Db>
    {
        private static DataSet<ProductCategory> MockData()
        {
            return DataSet<ProductCategory>.ParseJson(Strings.Mock_ProductCategory);
        }

        protected override void Initialize()
        {
            Mock(Db.ProductCategory, MockData);
        }
    }
}

File: samples/VisualBasic/AdventureWorksLT.DbDesign/ProductCategoryMockDb.vb
Imports DevZest.Data

Public Class ProductCategoryMockDb
    Inherits MockDb(Of Db)

    Private Shared Function MockData() As DataSet(Of ProductCategory)
        Return DataSet(Of ProductCategory).ParseJson(My.Strings.Mock_ProductCategory)
    End Function

    Protected Overrides Sub Initialize()
        Mock(Db.ProductCategory, Function() MockData())
    End Sub
End Class
